src/word.c

```diff
@@ -0,0 +1,67 @@
+#include "dictionary.h"
+#include <stdio.h>
+#include <strings.h>
+#include <string.h>
+#include <assert.h>
+#include <stdlib.h>	
+
+int num_punctuation = 22;
+char* punctuation_array[] = {"+",",",":"," ",".","-","'","&","!","?",":",";","#","~","=","/","$","£","^","\n","_","<",">"};
+
+// checks if punctuation exists within array
+int is_in_array(char* punctuation_array[], char* word) {
```

Comment:
Why are you trying to pass in a global variable in a function declaration?

Reply:
@sarikam1